visualization/app/codeCharta/util/customConfigHelper.ts

```diff
@@ -78,22 +95,11 @@ export class CustomConfigHelper {
 		return false
 	}
 
-	static getCustomConfigByName(
-		mapSelectionMode: CustomConfigMapSelectionMode,
-		selectedMaps: string[],
-		configName: string
-	): CustomConfig | null {
-		for (const customConfig of CustomConfigHelper.customConfigs.values()) {
-			if (
-				customConfig.name === configName &&
-				customConfig.mapSelectionMode === mapSelectionMode &&
-				customConfig.assignedMaps.join("") === selectedMaps.join("")
-			) {
-				return customConfig
-			}
+	private static areEqual(map1: MapNameByChecksum, map2: MapNameByChecksum) {
+		if (map1.size !== map2.size) {
+			return false
 		}
-
-		return null
+		return [...map1.entries()].every(([checksum, mapName]) => map2.has(checksum) && map2.get(checksum) === mapName)
```

Comment:
Lets assume `map1 = [{ key: "abc", value "a.cc.json" }]` and `map2 = [{ key: "abc", value "a.cc.json" }, { key: "def", value "d.cc.json" }]`: Is `areEqual(map1, map2)` supposed to be true? Currently it would be, although map2 has more entries.

Reply:
In the lines above I wrote this statement to prevent this. I think you missed that for sure :D
 `if (map1.size !== map2.size) {
			return false
		}`